### PR TITLE
Fix various typos in log messages

### PR DESCRIPTION
**Pull request by CounterPillow**
I tried to keep it strictly to things that are clearly a bit off, and not just odd style choices such as the RANDOM SHOUTING in the dvb code.

I agree that my changes can be relicensed to LGPL 2.1 or later.

### CODE REVIEW

File: demux/ebml.c [hunk 1]
```diff
@@ -509,8 +509,8 @@ static void ebml_parse_element(struct ebml_parse_ctx *ctx, void *target,
         bool multiple = fd->multiple;
         int *countptr = (int *) (s + fd->count_offset);
         if (*countptr >= num_elems[field_idx]) {
-            // Shouldn't happen with on any sane file without bugs
-            MP_ERR(ctx, "Too many subelems?\n");
+            // Shouldn't happen on any sane file without bugs
+            MP_ERR(ctx, "Too many subelements.\n");
```
Comment (avih) [inline demux/ebml.c]: "sub-elements" ?

Comment (CounterPillow) [reply demux/ebml.c]: "subelements" is used in other log messages too.

**Comment by timofonic** [comment]
@CounterPillow @avih @wm4 Any news?

**Comment by kevmitch** [comment]
@CounterPillow what's with the travis failure? Could you rebase and force push to kick it?

**Pull request merged**